radioapp/ui/ui_backlight.py

```diff
@@ -0,0 +1,62 @@
+from tkinter import *
+
+
+class UIBackLight():
+	"""Creates UI Backlight Screen."""
+
+	def __init__(self):
+		"""Creates a window, and adds slider and label for UI Backlight."""
+
+		self.window = Tk()
+		self.window.title("Backlight Setting")
+		self.window.geometry("160x128")
+		self.window.configure(background = "#ff9966")
+		self.window.resizable(0, 0)
+		self.window.pack_propagate(0)
+		
+		#Binds key presses to change slider value.
+		self.window.bind("<Up>", self.up_key)
+		self.window.bind("<Down>", self.down_key)		
+		
+		#Adds slider to the window.
+		self.add_slider()
+		
+		#Adds label to the window.
+		self.add_label()		
+		
+		self.window.mainloop()

```

Comment:
Same as above, the backlight screen doesn't need to run the main loop

Reply:
Yeah, I have to have to integrate UI Backlight to the MVP,  use manager classes and other classes, but MVP needs refactoring first so it's easily extensible. The MVP already has certain issues that needs to be fixed before new functionalities are added.

Reply:
Curious on what part of the MVP that needs to be refactored for this to work, the map already has an example of how to make a back button, and use its window/mainloop.